lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -56,8 +56,7 @@ def test_noscale():
 
     fig = plt.figure()
     ax = fig.add_subplot(1, 1, 1)
-    ax.imshow(Z, cmap='gray')
-    plt.rcParams['svg.image_noscale'] = True
+    ax.imshow(Z, cmap='gray', interpolation='none')
```

Comment:
I don't think it's worth the effort to keep the old rcParam working.  The user will get a warning that includes an easy workaround, however.

`svg.image_noscale` has been around since 2006 in 859f591c.  But it's replacement, `interpolation == 'none'` has been around since 2009-ish in 67134404 -ish.  If removing `svg.image_noscale` didn't have a super-easy workaround, I'd say we should invest effort into keeping it.  We *could* make it implicitly set `image.interpolation` to `none`, which would have the same effect, except it would apply to all backends.  I'm not sure that's any better, though.